src/SuperDumpService/Models/DumpMetainfo.cs

```diff
@@ -44,7 +44,10 @@ public enum DumpType {
 		WindowsDump,
 		
 		// *.core.gz    -> linux coredump file
-		LinuxCoreDump
+		LinuxCoreDump,
+
+		//Only show additional files if no Dump was found
+		Empty
```

Comment:
I see how we need that change now, but it's actually really unclean.

Longer term, I believe we need to re-architect SuperDump to make to the central entity not a "dump", but rather an "archive"/"bundle". Certain files from that archive might receive extra analysis (like a memory dumps of certain kinds), but otherwise SuperDump just shows the files from an archive.